core/templates/dev/head/components/state-editor/state-editor-properties-services/state-property.service.ts

```diff
@@ -15,75 +15,103 @@
 /**
  * @fileoverview Standalone services for the general state editor page.
  */
+import cloneDeep from 'lodash/cloneDeep';
```

Comment:
Please move it below the angular imports. Below line 21 with a line break above and below. We maintain angular imports right at the top :).

Reply:
Ok, that's strange. I see the reverse of this in almost all of the files. Should I fix them too?

Reply:
Umm, no you don't need to do that -- we didn't had any style guide then neither we have now but this is something that we are fixed with right now i.e angular imports should be at the top.

I'll change it at other places too :), thanks

Reply:
Done.